.travis.yml

```diff
@@ -52,6 +52,11 @@ install:
 # print hugo version (useful for debugging documentation build errors)
 - hugo version
 
+# dependencies for swimm
+- sudo apt-get update
+- sudo apt-get -y install libappindicator3-1
+- sudo apt-get -y install libsecret-1-0
```

Comment:
`sudo apt-get -f install` after `dpkg -i swimm` should be better, as we wouldn't have to figure out dependencies each time they change